contracts/src/management/ManagementContract.sol

```diff
@@ -114,20 +114,20 @@ contract ManagementContract is Initializable, OwnableUpgradeable {
         return isBundleSaved[bundleHash];
     }
 
-    function addCrossChainMessagesRoot(bytes32 _lastBatchHash, bytes32 blockHash, uint256 blockNum, bytes[] memory crossChainHashes, bytes calldata signature, uint256 rollupNumber, bytes32 forkID) external {
-        /*  if (block.number > blockNum + 255) {
-              revert("Block binding too old");
-          }
+    function blockBinding(uint256 blockNum) public view returns(bytes32) {
```

Comment:
is this used?

Reply:
Was used for debugging and figuring out if im crazy, will remove